composer.json

```diff
@@ -18,24 +18,25 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^8.0 || ^8.1",
-        "silverstripe/vendor-plugin": "^1",
-        "silverstripe/framework": "^4.11",
-        "silverstripe/admin": "^1.11",
-        "silverstripe/userforms": "^5.14",
-        "dnadesign/silverstripe-elemental": "^4.4",
-        "silverstripe/smime": "dev-master",
-        "dnadesign/silverstripe-elemental-userforms": "^3.2",
-        "silverstripe/asset-admin": "^1.11",
-        "unclecheese/display-logic": "^2.0",
-        "lekoala/silverstripe-encrypt": "^3.4"
+        "php": "^8.1",
+        "silverstripe/vendor-plugin": "~2.0",
+        "silverstripe/framework": "~5.2",
```

Comment:
@pjayme Is there any reason why this needs to be locked to CMS 5.2 or above? For example, if I have CMS 5.1 environment but I won't be able to add this to my project?

Reply:
Good catch @adunn49 have reverted to `^5` instead

Reply:
@pjayme You will also need to update the `silverstripe/framework` to match (otherwise still can't install on CMS 5.1, not that it is an issue for us) and also change the silverstripe reference in the README. To be honest, I am not sure why I've included both framework and cms dependencies, or whether there was a reason for that 🤔

Reply:
@adunn49 my bad there, have also updated `silverstripe/framework` and as discussed, removed the `~` constraint in favour of `^` to prevent future versions from breaking